packages/protocol-kit/src/types/index.ts

```diff
@@ -159,7 +159,7 @@ export interface SwapOwnerTxParams {
 
 type StandardizeSafeTxDataWithSafeContractProps = {
   /** safeContract - The Safe contract to use */
-  safeContract: SafeContract
+  safeContract: SafeContactType
```

Comment:
There is a typo here. Review the code where this parameter is used
```suggestion
  safeContract: SafeContractType
```